pkg/stanza/operator/transformer/flatten/testdata/config.yaml

```diff
@@ -1,6 +1,9 @@
 flatten_attributes:
   type: flatten
-  field: attributes.errField
+  field: attributes.secondlevel
```

Comment:
This is very minor, but elsewhere we use "secondlevel" to indicate a second level of nesting, so perhaps we can have tests for `attributes.nested`, and `attributes.nested.secondlevel`.